geom/util_test.go

```diff
@@ -168,6 +174,25 @@ func expectBytesEq(t testing.TB, got, want []byte) {
 	}
 }
 
+func expectFloat64Eq(t testing.TB, got, want float64) {
+	t.Helper()
+	if got != want {
+		t.Errorf("\ngot:  %v\nwant: %v\n", got, want)
+	}
+}
+
+func expectEnvEq(t testing.TB, got, want Envelope) {
+	t.Helper()
+	if !ExactEquals(got.Min().AsGeometry(), want.Min().AsGeometry()) ||
+		!ExactEquals(got.Max().AsGeometry(), want.Max().AsGeometry()) {
```

Comment:
nit: I would invert this to: 
```
if ExactEquals(...) && ExactEquals(...) { 
    return 
}
t.Errorf(...)
```

Reply:
Good suggestion, that's much easier to read. Changed in 3d837b732f6c1083fdb7857da3f526f8b30c2d8c